### ISSUE TITLE
removeEventListener is broken for passive events

### ISSUE THREAD

**Issue by zorcec** [angular/zone.js#917]
How to replicate. Try to add a passive event.
e.q.
function test() {
 console.log("test");
}

document.addEventListener('scroll', test, {passive: true});

Try to remove it:
document.removeEventListener('scroll', test, {passive: true});

This happens on every website where ng2+ is loaded.

**Comment by JiaLiPassion** [comment]
@zorcec , I can't reproduce, could you provide a reproduce plunker or repo? thanks

**Comment by zorcec** [comment]
I retried the following:
 - zone.js latest only 
 -  ng4 latest

In both cases, I was not able to reproduce it.
ng4 e.q.
https://embed.plnkr.co/IK9X5w/

But if you try older version (e.q. 4.1.3) like it is in this example: https://www.poetrybuddy.com/
You will experience the problem.

Just make the page scroll-able and execute the following code in the console:
```
var scrollHandler = function() {
  console.log('test');
};
document.addEventListener('scroll', scrollHandler, { passive: true });
```

Remove will not work:
`document.removeEventListener('scroll', scrollHandler, { passive: true });`

My conclusion is that this was fixed in the newer versions.
If you have any tip how this can be fixed without ng update I would be grateful.
Thank you for your time.
